modules/launcher/src/main/java/org/mule/runtime/module/launcher/log4j2/ArtifactAwareContextSelector.java

```diff
@@ -47,7 +49,8 @@
  */
 class ArtifactAwareContextSelector implements ContextSelector, Disposable {
 
-  static final StatusLogger logger = StatusLogger.getLogger();
+  static final StatusLogger STATUS_LOGGER = StatusLogger.getLogger();
```

Comment:
Any suggestion? It was simply "logger" before..  I'm not 100% clear on what role of this static logger is.